cms/pytest.ini

```diff
@@ -8,10 +8,17 @@ addopts = --nomigrations --reuse-db --durations=20 -p no:randomly --json-report
 # Enable default handling for all warnings, including those that are ignored by default;
 # but hide rate-limit warnings (because we deliberately don't throttle test user logins)
 # and field_data deprecation warnings (because fixing them requires a major low-priority refactoring)
+# ABC deprecation Warning comes from libsass
+# declare_namespace Warning comes from XBlock https://github.com/openedx/XBlock/issues/641
```

Comment:
We should add this link next to the specific warning. Otherwise, it will likely be omitted during the removal.

Reply:
will do that. trailing comments throw errors, but i'll add them next to their corresponding warnings